modelforge/potential/tensornet.py

```diff
@@ -96,10 +99,17 @@ class TensorNetRepresentation(torch.nn.Module):
     def __init__(
         self,
         radial_max_distance,
-        radial_min_distanc,
+        radial_min_distance,
         number_of_radial_basis_functions,
     ):
-        pass
+        super().__init__()
+
+        self.cutoff_module = CosineCutff(radial_max_distance)
```

Comment:
The reason for the error in gh action is a typo here --- it should be `CosineCutoff`